Libplanet.Explorer.Executable/Program.cs

```diff
@@ -38,7 +38,11 @@ public static void Main(string[] args)
                 .WriteTo.Console();
             Log.Logger = loggerConfig.CreateLogger();
 
-            IStore store = new LiteDBStore(options.StorePath, readOnly: options.Seed is null);
+            IStore store = new LiteDBStore(
+                path: options.StorePath,
+                flush: false,
```

Comment:
It's intended to improve I/O performance on Azure in particular, right?

Reply:
Yes.